comparsion-and-logical-operators/3.grading-system.js

```diff
@@ -0,0 +1,17 @@
+var gradeScore = 61;
+
+if (gradeScore < 40) {
```

Comment:
Grade is undeclared: need to do this before the if statement.
Can set to 'A' and you don't need the final comparison.